charts/metallb/templates/webhooks.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: validating-webhook-configuration
+  name: metallb-webhook-configuration
```

Comment:
Should this be templated? `{{ template "metallb.fullname" . }}`. Or is it always hardcoded to `metallb-webhook-configuration` in the code?

Reply:
It's hardcoded here https://github.com/metallb/metallb/pull/1497/files#diff-14c68fe5b25ee985e2af6d5dc8c7eaef1b4d91f0dac32019bf971d5681c803beR62 (that's needed for the cert-controller to patch the webhooks), so I guess it's fine as it is.
If we want to make it configurable, we should also change the controller to accept it as a parameter, but I am not sure it's worth the effort.